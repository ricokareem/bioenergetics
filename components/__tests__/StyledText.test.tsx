import * as React from 'react';
import { shallow } from 'enzyme';
import { MonoText } from '../StyledText';

describe('<MonoText>', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<MonoText>Snapshot test!</MonoText>);
      expect(component).toMatchSnapshot();
    });
  });
});
