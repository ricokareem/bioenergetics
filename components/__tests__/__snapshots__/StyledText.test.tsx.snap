// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MonoText> Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
